### PR TITLE
update timesync tables

### PR DESCRIPTION
**Pull request by c11**
1. change timesync related table names using CEO naming convention.
2. remove obsolete table and functions.

this upate only affect database structure. No functional implementation for timesync is included in this pull request.

### CODE REVIEW

**Comment by sirmspencer** [comment]
Can you update your tables to match as well?
Rules are
1.  primary key is named after table `table comments` -> `comment_uid` (singular)
2.  foreign key match the primary key but use `_rid`
3.  for function inputs start with `_` so its easy to tell in the function what is an input value
4.  for function inputs that = a table column have them match (except the _) `where comment_uid = _comment_uid`
5.  for function return values use only `_id`

**Comment by c11** [comment]
I will put this edits using the rules you specified. However, I do not
understand rule 5: for function return values use only _id

File: src/main/resources/sql/load_ts_tables.sql [hunk 1]
```diff
@@ -1,59 +1,48 @@
...
-
-DROP TABLE IF EXISTS ts_plot_comments cascade;
-CREATE TABLE ts_plot_comments (
-  id                    bigserial primary key,
-  project_id            integer not null references projects(id) on update cascade,
-  plot_id               integer not null references user_plots (id) on delete cascade on update cascade,
-  interpreter           integer not null references users (id) on update cascade,
+DROP TABLE IF EXISTS plot_comments cascade;
+CREATE TABLE plot_comments (
+  plot_comments_uid           bigserial primary key,
```
Comment (sirmspencer) [inline src/main/resources/sql/load_ts_tables.sql]: plot_comment_uid

Comment (sirmspencer) [reply src/main/resources/sql/load_ts_tables.sql]: fix spacing after plot_comment_uid (align column for data type)

File: src/main/resources/sql/load_ts_tables.sql [hunk 2]
```diff
@@ -1,59 +1,48 @@
 -- TS related tables
-DROP TABLE IF EXISTS ts_packets cascade;
-CREATE TABLE ts_packets (
-	id             serial primary key,
-    project_id     integer not null references projects(id) on delete cascade on update cascade,
-	packet_id      integer not null,
-    plot_id        integer not null references user_plots (id) on delete cascade on update cascade
+DROP TABLE IF EXISTS packets cascade;
+CREATE TABLE packets (
+  packet_id     integer not null,
```
Comment (sirmspencer) [inline src/main/resources/sql/load_ts_tables.sql]: packet_uid

File: src/main/resources/sql/load_ts_tables.sql [hunk 3]
```diff
@@ -1,59 +1,48 @@
...
+    landcover           text NOT NULL,
+    change_process      text NOT NULL
+);
+DROP INDEX IF EXISTS prjid;
+CREATE UNIQUE INDEX prjid on response_design(project_rid);
+
+
+DROP TABLE IF EXISTS vertex cascade;
+CREATE TABLE vertex (
   vertex_id                 bigserial PRIMARY KEY,
```
Comment (sirmspencer) [inline src/main/resources/sql/load_ts_tables.sql]: vertex_uid

File: src/main/resources/sql/load_ts_tables.sql [hunk 4]
```diff
@@ -69,27 +58,28 @@ CREATE TABLE ts_vertex (
   change_process            varchar(30) DEFAULT NULL,
   change_process_notes      varchar(255) DEFAULT NULL,
   comments                  varchar(255) DEFAULT NULL,
-  interpreter               integer not null references users (id) on update cascade,
+  user_rid               integer not null references users (user_uid) on update cascade,
```
Comment (sirmspencer) [inline src/main/resources/sql/load_ts_tables.sql]: fix alignment of data type

File: src/main/resources/sql/load_ts_tables.sql [hunk 5]
```diff
@@ -69,27 +58,28 @@ CREATE TABLE ts_vertex (
...
   last_modified             timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
   history_flag              integer DEFAULT 0,
-  packet_id                 integer DEFAULT -1
+  packet_id                integer DEFAULT -1 --references packets(packet_id) on delete cascade on update cascade
```
Comment (sirmspencer) [inline src/main/resources/sql/load_ts_tables.sql]: fix alignment of data type

File: src/main/resources/sql/load_ts_tables.sql [hunk 6]
```diff
@@ -69,27 +58,28 @@ CREATE TABLE ts_vertex (
...
+CREATE TABLE image_preference (
+    image_preference_uid  serial primary key,
+    project_rid           integer not null references projects(project_uid) on delete cascade on update cascade,
+    plot_rid              integer not null,
+    image_id              text,
+    image_year            integer not null,
+    image_julday          integer not null,
+    user_rid              integer not null references users (user_uid) on update cascade,
+    "priority"            integer not null,
+    packet_id            integer default -1
```
Comment (sirmspencer) [inline src/main/resources/sql/load_ts_tables.sql]: fix alignment of data type

File: src/main/resources/sql/load_ts_tables.sql [hunk 7]
```diff
@@ -1,59 +1,48 @@
...
+  plot_rid              integer not null references plots(plot_uid) on delete cascade on update cascade,
+  user_rid              integer not null references users(user_uid) on update cascade,
   comment               text,
   is_example            integer DEFAULT NULL,
   is_complete           integer DEFAULT NULL,
   is_wetland            integer DEFAULT NULL,
   uncertainty           integer DEFAULT NULL,
   last_modified_date    timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
-  packet_id             integer default -1
+  packet_id             integer default -1 --references packets(packet_id) on delete cascade on update cascade
```
Comment (sirmspencer) [inline src/main/resources/sql/load_ts_tables.sql]: if this is going to reference packet_uid, change to packet_rid

File: src/main/resources/sql/load_ts_tables.sql (hunk 5 above)
Comment (sirmspencer) [reply src/main/resources/sql/load_ts_tables.sql]: if this is going to reference packet_uid, change to packet_rid

File: src/main/resources/sql/load_ts_tables.sql (hunk 7 above)
Comment (c11) [reply src/main/resources/sql/load_ts_tables.sql]: changed based on yesterday and today's discussion.

**Comment by sirmspencer** [comment]
user_plots->packet_rid
--- Since we are not planning on rebuilding our DB, this needs to be done as an alter statement after the packets table is created.

**Comment by c11** [comment]
Modification made following suggestions from requested changes.

**Pull request merged**